### PR TITLE
Add MDN instructions to update email.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

**Comment by jpmedley** [comment]
@ebidel Any feedback on this? I'd like to wrap this up before I start my xmas vacation at 5:00 PM PST, Thursday.

File: notifier.py [hunk 1]
```diff
@@ -84,6 +84,13 @@ def list_diff(subscribers, owners):
     else:
       milestone_str = 'not yet assigned'
 
+	moz_steps = ''
+	for link in feature.doc_links:
+	  if 'developer.mozilla.org' in link:
+	    moz_links += ('<li>' + link + '</li>')
```
Comment (ebidel) [inline notifier.py]: where does `moz_links` get defined? This is the first time you're using it.

File: notifier.py [hunk 2]
```diff
@@ -161,7 +169,7 @@ def list_diff(subscribers, owners):
            owners=', '.join([o.name for o in owners]), milestone=milestone_str,
            status=models.IMPLEMENTATION_STATUS[feature.impl_status_chrome],
            formatted_changes=formatted_changes,
-           wf_content=create_wf_content_list(component_name),
+           wf_content=create_wf_content_list(component_name),moz_steps=moz_steps,
```
Comment (ebidel) [inline notifier.py]: space after `,`

File: notifier.py (hunk 1 above)
Comment (ebidel) [reply notifier.py]: `moz_links += '<li>%s</li>' % link`

File: notifier.py [hunk 3]
```diff
@@ -84,6 +84,13 @@ def list_diff(subscribers, owners):
     else:
       milestone_str = 'not yet assigned'
 
+	moz_steps = ''
+	for link in feature.doc_links:
+	  if 'developer.mozilla.org' in link:
+	    moz_links += ('<li>' + link + '</li>')
+	if moz_links:
+		moz_steps = '<li>Review the following MDN pages and <a href="https://docs.google.com/document/d/10jDTZeW914ahqWfxwm9_WXJWvyAKT6EcDIlbI3w0BKY/edit#heading=h.frumfipthu7">subscribe to updates</a> for them.<ul>' + moz_links + '</ul><li>'
```
Comment (ebidel) [inline notifier.py]: `</ul><li>` -> `</ul></li>`

File: notifier.py [hunk 4]
```diff
@@ -84,6 +84,13 @@ def list_diff(subscribers, owners):
     else:
       milestone_str = 'not yet assigned'
 
+	moz_steps = ''
```
Comment (jpmedley) [inline notifier.py]: @ebidel This is the definition. Is there some other way I should have done this?

Comment (ebidel) [reply notifier.py]: I think you can just declare `moz_links  = ''` replace all uses of `moz_steps` with `moz_links`. IOW, you don't need two variables.

File: notifier.py [hunk 5]
```diff
@@ -161,7 +169,7 @@ def list_diff(subscribers, owners):
            owners=', '.join([o.name for o in owners]), milestone=milestone_str,
            status=models.IMPLEMENTATION_STATUS[feature.impl_status_chrome],
            formatted_changes=formatted_changes,
-           wf_content=create_wf_content_list(component_name),
+           wf_content=create_wf_content_list(component_name), moz_steps=moz_steps,
```
Comment (ebidel) [inline notifier.py]: and moz_links here too

File: notifier.py (hunk 4 above)
Comment (ebidel) [reply notifier.py]: I think indentation is off on these lines. The line of your first change should align with the `intro` var.

Comment (jpmedley) [reply notifier.py]: Apologies. I didn't intend there to be two variables. I changed the name some point.

**Comment by jpmedley** [comment]
Back to you.

Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

**Comment by ebidel** [comment]
Did you see:

> I think indentation is off on these lines. The line of your first change should align with the intro var.

**Pull request merged**